src/idmodels/gbqr.py

```diff
@@ -165,7 +163,7 @@ def _train_gbq_and_predict(self, run_config,
         
         # melt to get columns into rows, keeping only the things we need to invert data
         # transforms later on
-        cols_to_keep = ["source", "location", "wk_end_date", "pop",
+        cols_to_keep = ["source", "agg_level", "location", "wk_end_date", "pop",
```

Comment:
will agg_level be present in datasets that are not from NSSP? and if not, will that be a problem for this code?